cms/plugin_base.py

```diff
@@ -386,22 +391,23 @@ def get_child_plugin_candidates(cls, slot, page):
         from cms.plugin_pool import plugin_pool
         return plugin_pool.get_all_plugins()
 
-    def get_child_classes(self, slot, page):
+    @classmethod
+    def get_child_classes(cls, slot, page):
```

Comment:
As with the one below, two reasons:

1. Consistency with the other `get_x` methods related to plugin classes/types.
2. It's much more efficient to call `Class.get_child_classes()` vs creating instance of `Class` every time just to call `get_child_classes`.